source/fundamentals/data-formats/codecs.txt

```diff
@@ -46,3 +46,30 @@ Custom Codec Example
 - Sample query to demonstrate decode results (of a field in a POJO) using custom codec
 - Sample insert and resulting fields of the document to demonstrate encode 
 - Example showing how to read/encode a document with unknown fields
+
```

Comment:
Accidental commit of some notes -- there's a ticket to create the content for this page in another PR.

Reply:
So is this PR contingent on that other PR?

Reply:
They're just notes that will get replaced by actual content in that PR.